src/Login/LoginStyles/login.scss

```diff
@@ -0,0 +1,107 @@
+
+@import '../../commons/assets/variables/font.scss';
+@import '../../commons/assets/variables/colors.scss';
+
+.Login-container{
+  margin: 10px 0 20px 0;
+
+  .myform{
+  width: 517px;
+  height: 480px;
+  border: #cccccc solid 1px;
+  margin-top: 30px;
+  margin-left: auto;
+  margin-right: auto;
+  padding-top: 70px;
+  }
+
+  .heading {
+    width: 387px;
+    height: 30px;
+    margin-left: auto;
+    margin-right: auto;
+    text-align: center;
+    opacity: 99%;
+  }
+  .form-input input, button{
+    margin-top: 30px;
+    width: 297px;
+    margin-left: 99px;
+    height: 40px;
+    padding: 6px 6px 6px 10px;
+    font-size: 14px;
+
+  }
+  .mypara{
+    color:#606060;
+    text-align: left;
+    font-family: 'Source Sans Pro';
```

Comment:
@jennizalwango set this font-family to `$source-san-font` variable instead.

Reply:
this is done